app/Http/Controllers/Teams/LeaveTeamController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Http\Controllers\Teams;
+
+use App\Http\Controllers\Controller;
+use App\Models\Teams\Team;
+use App\Models\User\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class LeaveTeamController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function __invoke(Request $request)
```

Comment:
Comment missing. What does this do?

Reply:
You mean the controller as a whole or the invoke method? If you mean the invoke method, no need to comment it, they're pretty common, see https://laravel.com/docs/8.x/controllers#single-action-controllers